app/src/main/java/org/openimis/imispolicies/OverViewPolicies.java

```diff
@@ -599,7 +603,7 @@ public void run() {
 
                 HttpResponse response = null;
                 try {
-                    response = toRestApi.postToRestApiToken(order, "payment/GetControlNumber");
+                    response = httpClient.execute(httpPost);
```

Comment:
Here we should get the paymentID instead of the controlNumber,

then in offline or online upload we should add the payment in the upload so the server can link the payment with the control number

![image](https://user-images.githubusercontent.com/4052505/118273721-94ebc080-b4c4-11eb-8eaa-c53384e0919f.png)

Reply:
On Monday I will get to know with that schema. Thx for doing this. For sure it helps me to understand more what you wanted!

Reply:
@delcroip The endpoint name might induce in error. Is called GetControlNumber but is not actually gives the CN as it is async call. Same approach as for the USSD: `GetControlNumber/Single`

Reply:
@delcroip I think this requested change might be develop in Damian's tickets OTC-264 https://openimis.atlassian.net/browse/OTC-264. I updated part of code related to the check build for switching path to /restapi for getControlNumber. See above. Then we can close this PR if everything is ok.